folly/experimental/exception_tracer/ExceptionTracer.cpp

```diff
@@ -200,6 +201,7 @@ std::vector<ExceptionInfo> getCurrentExceptions() {
   return exceptions;
 }
 
+#if !defined(__clang__)
```

Comment:
Why doesn't this work with clang?

Reply:
as mentioned in the comment, there is obsolete C++ 14 usage (e.g. std::unexpected_handler)

Reply:
Do you mean libc++ instead of clang?

Reply:
yes good point forgot FreeBSD clang uses libc++ whatsoever.

Reply:
Check for libstdc++ instead of !clang?